### DISCUSSION TITLE
How to set some more config items for Nginx in DOM Cloud instance using the Deployment Script?

### DISCUSSION THREAD

**Discussion by ghost** [domcloud/domcloud#65] [General]
For example, I need to set nginx like this:
`
    location / {
        proxy_set_header User-Agent $http_user_agent;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header Accept-Encoding "";
        proxy_set_header Accept-Language $http_accept_language;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    }
`

Any comments are welcome.

**Comment by willnode** [comment]
Should be possible using [passenger_set_header](https://www.phusionpassenger.com/docs/references/config_reference/nginx/#passenger_set_header) my bad it isn't supported yet as of now.

Please wait until this gets implemented. shouldn't take too long!
